### PR TITLE
Update daikin.markdown

### PR DESCRIPTION
**Pull request by emufan**
## Proposed change

Clarify/correct the description of the source of two entities. According to https://github.com/home-assistant/core/issues/72545

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- This PR fixes or closes issue: fixes https://github.com/home-assistant/core/issues/72545

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/daikin.markdown [hunk 1]
```diff
@@ -117,7 +117,7 @@ The `daikin` sensor platform integrates Daikin air conditioning systems into Hom
 
 <div class='note'>
 
-- The 'Today's total energy consumption' and 'Estimated power consumption' sensor is updated every time 100 Wh are consumed by all different operating modes summed together.
+- The 'Today's total energy consumption' and 'Estimated power consumption' sensor is updated every time 100 Wh are consumed by all different operating modes summed together. This values are the values for the whole installation, not per device (unless, you have onle one device).
```
Comment (cremor) [inline source/_integrations/daikin.markdown]: Typo in "onle".

Comment (emufan) [reply source/_integrations/daikin.markdown]: Fixed.

File: source/_integrations/daikin.markdown [hunk 2]
```diff
@@ -117,7 +117,7 @@ The `daikin` sensor platform integrates Daikin air conditioning systems into Hom
 
 <div class='note'>
 
-- The 'Today's total energy consumption' and 'Estimated power consumption' sensor is updated every time 100 Wh are consumed by all different operating modes summed together.
+- The 'Today's total energy consumption' and 'Estimated power consumption' sensor is updated every time 100 Wh are consumed by all different operating modes summed together. This values are the values for the whole installation, not per device (unless, you have only one device).
```
Comment (mlemainque) [inline source/_integrations/daikin.markdown]: Typo in "This", should be "These"

**Review by mlemainque — CHANGES_REQUESTED** [review]

File: source/_integrations/daikin.markdown (hunk 2 above)
Comment (emufan) [reply source/_integrations/daikin.markdown]: Fixed.

**Review by mlemainque — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @emufan 👍

**Pull request merged**